lib/manager/kustomize/extract.ts

```diff
@@ -122,11 +122,10 @@ export function parseKustomize(content: string): Kustomize | null {
     return null;
   }
 
-  pkg.bases = (pkg.bases || []).concat(
-    pkg.resources || [],
-    pkg.components || []
+  pkg.resources = (pkg.resources ?? []).concat(
+    pkg.bases ?? [], // bases is deprecated since Kustomize v2.1.0
+    pkg.components ?? []
   );
```

Comment:
i like to completly avoid this manipulation here and simply add multiple `for ... of ...` loops to extract function.